lib/features/mailbox/domain/extensions/mailbox_name_extension.dart

```diff
@@ -1,5 +1,10 @@
 import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
 
 extension MailboxNameExtension on MailboxName {
-  int compareToSort(MailboxName mailboxName) => name.compareTo(mailboxName.name);
+  int compareAlphabetically(MailboxName? other) {
+    if (other == null) {
+      return 1;
```

Comment:
why return `1` not `0`

Reply:
if `other` is null, it should be less than `node`